main.py

```diff
@@ -1,10 +1,56 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, other_info in data.items():
+
+        player_email = other_info["email"]
+        player_bio = other_info["bio"]
+        player_race = other_info["race"]
+        player_guild = other_info["guild"]
+        player_skills = player_race["skills"]
+
+        if not Race.objects.filter(name=player_race["name"]).exists():
+            Race.objects.create(
+                name=player_race["name"],
+                description=player_race["description"]
+            )
+        player_race = Race.objects.get(name=player_race["name"])
```

Comment:
can you optimize it? At least, you can make `race = Race.objects.create()...`. 
Then, you don't have to make an additional request to the database to get that value
Now the difference is not felt much, but in real projects, it is VERY important.

Reply:
Thanks! I've read about get_or_create after finishing the task, so I used it now! :)